lib/app.js

```diff
@@ -1,11 +1,9 @@
-'use strict';
-
-const R = require('ramda');
-const queryString = require('querystring');
-const request = require('./utils/request');
-const scriptData = require('./utils/scriptData');
-const { BASE_URL } = require('./constants');
-const helper = require('./utils/mappingHelpers');
+import { assoc, path } from 'ramda';
```

Comment:
I noticed that you replaced a lit of module imports eg R for direct imports of module functions. Please don't do that, as we lose the information of where each function is coming from, making the code less readable